tests/features/background.feature

```diff
@@ -4,7 +4,7 @@ Feature: Simple Background
     The Background shall be assigned to
     each Scenario.
 
-    Background: A simple Background
+    Background:
```

Comment:
It would be great to have an additional test feature `tests/features/background-no-sentence.feature` to test both scenarios: one with the sentence / summary and one without. 

You would have to add an additional test, too.